lib/load.php

```diff
@@ -128,3 +128,120 @@ function gutenberg_is_experiment_enabled( $name ) {
 require __DIR__ . '/block-supports/typography.php';
 require __DIR__ . '/block-supports/custom-classname.php';
 require __DIR__ . '/block-supports/border.php';
+
+
+// Demo query patterns.
+register_block_pattern_category(
+	'core/query',
+	array( 'label' => __( 'Core Query block', 'gutenberg' ) )
```

Comment:
I think the patterns should be called "Post List" or something to that effect

Reply:
These are just demo patterns that will not be included in the final code.